src/bio/bio_xstream.c

```diff
@@ -1716,6 +1716,34 @@ scan_bio_bdevs(struct bio_xs_context *ctxt, uint64_t now)
 	nvme_glb.bd_scan_age = now;
 }
 
+void
+bio_led_event_monitor(struct bio_xs_context *ctxt, uint64_t now)
+{
+	struct bio_bdev         *d_bdev;
+	struct spdk_bdev        *bdev;
+	static uint64_t          led_event_period = NVME_MONITOR_PERIOD;
+
+	for (bdev = spdk_bdev_first(); bdev != NULL;
+	     bdev = spdk_bdev_next(bdev)) {
+		d_bdev = lookup_dev_by_name(spdk_bdev_get_name(bdev));
+		if (d_bdev == NULL)
+			continue;
```

Comment:
Above lookup isn't necessary if you iterate bio_bdev list directly.

Reply:
done